src/main/java/com/unzer/payment/Unzer.java

```diff
@@ -195,6 +195,18 @@ public Authorization authorize(BigDecimal amount, Currency currency, String type
                 getAuthorization(amount, currency, typeId, returnUrl, customerId, card3ds, null));
     }
 
+    /**
```

Comment:
Can you check if this documentation is correct here ?

Reply:
Updated description to preauth.